.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,5 @@
-### All Submissions:
+DO NOT CREATE NEW PULL REQUESTS ON THIS REPOSITORY!
```

Comment:
People should be reading this in the new pull request where it will not be formatted.